sources/install/package_base.sh

```diff
@@ -199,6 +199,16 @@ function install_gf() {
     add-to-list "gf,https://github.com/tomnomnom/gf,A wrapper around grep to avoid typing common patterns"
 }
 
+function install_cyberchef() {
+    colorecho "Installing CyberChef"
+    mkdir /opt/tools/CyberChef
+    LAST_VERSION=$(wget https://github.com/gchq/CyberChef/releases/latest/download/CyberChef.zip 2>&1 | grep Location: | grep -E -o 'v.*' | cut -d '/' -f 1)
+    wget https://github.com/gchq/CyberChef/releases/download/$LAST_VERSION/CyberChef_$LAST_VERSION.zip -O /tmp/CyberChef.zip
+    unzip /tmp/CyberChef.zip -d /opt/tools/CyberChef/
```

Comment:
After unzipping, could you remove the file from /tmp ? Probably not indeed due to its location, but it's better to clean up explicitely in my opinion